### PR TITLE
Katello 3.7 RC docs tweaks

### PR DESCRIPTION
**Pull request by jturel**
Removed f26 to accurately reflect our repos and other small things

### CODE REVIEW

**Comment by jturel** [comment]
@akofink does this reflect what you mentioned to me on IRC?

File: plugins/katello/3.7/installation/clients.md [hunk 1]
```diff
@@ -19,7 +19,6 @@ Install the appropriate Katello client release packages.  For CentOS 6, you will
      <option value="el5">Enterprise Linux 5 (CentOS, etc.)</option>
      <option value="el6">Enterprise Linux 6 (CentOS, etc.)</option>
      <option value="el7">Enterprise Linux 7 (CentOS, etc.)</option>
-     <option value="f26">Fedora 26</option>
      <option value="f27">Fedora 27</option>
```
Comment (akofink) [inline plugins/katello/3.7/installation/clients.md:22]: Add F28

File: plugins/katello/3.7/installation/clients.md [hunk 2]
```diff
@@ -46,12 +45,6 @@ yum -y install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch
 {% endhighlight %}
 </div>
 
-<div id="f26" style="display:none;" markdown="1">
-{% highlight bash %}
-yum install -y http://fedorapeople.org/groups/katello/releases/yum/{{ page.version }}/client/f26/x86_64/katello-client-repos-latest.rpm
-{% endhighlight %}
-</div>
-
```
Comment (akofink) [inline plugins/katello/3.7/installation/clients.md:54]: And here.

**Review by akofink — COMMENTED** [review]
Almost!

File: plugins/katello/3.7/installation/clients.md (hunk 1 above)
Comment (jturel) [reply plugins/katello/3.7/installation/clients.md:22]: We don't have F28 client bits

edit... some kind of koji issue so we had to do without

**Comment by akofink** [comment]
Oh I see. Maybe we should delete https://fedorapeople.org/groups/katello/releases/yum/3.7/client/f28/ ?

**Review by akofink — APPROVED** [review]
Looks good!

**Comment by jturel** [comment]
Thanks! Probably a good idea on the empty repo removal...

**Pull request merged**